### PR TITLE
Enabling Comment Spacing StyleCop Rules

### PR DESCRIPTION
**Pull request by peterfreiling**

### CODE REVIEW

File: Sources/Core/Microsoft.StreamProcessing/Operators/Afa/AfaInterfaces.cs [hunk 1]
```diff
@@ -143,62 +159,75 @@ public interface IAbstractPatternRoot<TKey, TPayload, TRegister, TAccumulator>
...
+
         /// <summary>
         /// Interface is only public to serve CLR strong typing - instances should only be created internally. Do not use directly.
         /// </summary>
         /// <param name="condition"></param>
         /// <param name="aggregator"></param>
         /// <returns></returns>
         IPattern<TKey, TPayload, TRegister, TAccumulator> ListElement(Expression<Func<List<TPayload>, bool>> condition, Expression<Func<List<TPayload>, TRegister>> aggregator);
+
+
```
Comment (cybertyche) [inline Sources/Core/Microsoft.StreamProcessing/Operators/Afa/AfaInterfaces.cs]: Why the double spaces? Let's standardize on only single spaces if possible.

File: Sources/Core/Microsoft.StreamProcessing/Operators/Afa/AfaInterfaces.cs [hunk 2]
```diff
@@ -225,13 +254,16 @@ public interface IAbstractPattern<TKey, TPayload, TRegister, TAccumulator> : IAb
...
         /// <summary>
         /// Interface is only public to serve CLR strong typing - instances should only be created internally. Do not use directly.
         /// </summary>
         /// <typeparam name="TRegisterNew"></typeparam>
         /// <param name="defaultRegister"></param>
         /// <returns></returns>
         IAbstractPattern<TKey, TPayload, TRegisterNew, TAccumulator> SetRegister<TRegisterNew>(TRegisterNew defaultRegister = default);
+
+
```
Comment (cybertyche) [inline Sources/Core/Microsoft.StreamProcessing/Operators/Afa/AfaInterfaces.cs]: Here too

File: Sources/Core/Microsoft.StreamProcessing/Operators/Afa/AfaInterfaces.cs [hunk 3]
```diff
@@ -143,62 +159,75 @@ public interface IAbstractPatternRoot<TKey, TPayload, TRegister, TAccumulator>
...
         /// <summary>
         /// Interface is only public to serve CLR strong typing - instances should only be created internally. Do not use directly.
         /// </summary>
         /// <param name="condition"></param>
         /// <param name="aggregator"></param>
         /// <returns></returns>
         IPattern<TKey, TPayload, TRegister, TAccumulator> ListElement(Expression<Func<List<TPayload>, TRegister, bool>> condition, Expression<Func<List<TPayload>, TRegister, TRegister>> aggregator);
+
+
```
Comment (cybertyche) [inline Sources/Core/Microsoft.StreamProcessing/Operators/Afa/AfaInterfaces.cs]: And so on

File: Sources/Core/Microsoft.StreamProcessing/Operators/Afa/UngroupedDAfaTemplate.cs [hunk 4]
```diff
@@ -14,7 +14,7 @@ namespace Microsoft.StreamProcessing
     /// <summary>
     /// Class to produce the template output
     /// </summary>
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "16.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "15.0.0.0")]
```
Comment (cybertyche) [inline Sources/Core/Microsoft.StreamProcessing/Operators/Afa/UngroupedDAfaTemplate.cs]: Looks like someone hasn't upgraded to Visual Studio 2019 yet? :-)

File: Sources/Core/Microsoft.StreamProcessing/StringHandling/MultiString.cs [hunk 5]
```diff
@@ -263,9 +267,8 @@ public static unsafe void AndEquals(ColumnBatch<long> left, ColumnBatch<long> ri
 
         #endregion
 
-        // Clone the multi-string shell only
         /// <summary>
-        /// Currently for internal use only - do not use directly.
+        /// Clone the multi-string shell only
```
Comment (cybertyche) [inline Sources/Core/Microsoft.StreamProcessing/StringHandling/MultiString.cs]: There are a few places here where the "do not use directly" comment is replaced by a different comment. I think the "do not use directly" directive doesn't change here - we're saying that this is a public method that we don't want public that that people shouldn't count on it being around. Perhaps combine the two comments?

File: Sources/Core/Microsoft.StreamProcessing/Operators/Afa/AfaInterfaces.cs (hunk 1 above)
Comment (peterfreiling) [reply Sources/Core/Microsoft.StreamProcessing/Operators/Afa/AfaInterfaces.cs]: Accidental - Find&Replace mistake

---
In reply to: [286226139](https://github.com/microsoft/Trill/pull/94#discussion_r286226139) [](ancestors = 286226139)

File: Sources/Core/Microsoft.StreamProcessing/Operators/Afa/UngroupedDAfaTemplate.cs (hunk 4 above)
Comment (peterfreiling) [reply Sources/Core/Microsoft.StreamProcessing/Operators/Afa/UngroupedDAfaTemplate.cs]: I'll start it now, unless you think it's needed for this change

---
In reply to: [286226689](https://github.com/microsoft/Trill/pull/94#discussion_r286226689) [](ancestors = 286226689)

Comment (cybertyche) [reply Sources/Core/Microsoft.StreamProcessing/Operators/Afa/UngroupedDAfaTemplate.cs]: Nah, just for the future.

**Pull request merged**